common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/core/validation/InetAddressValidator.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Target platform can be modified later, then just few lines have to be removed from pom.xml, .gitignore and .classpath. 
With copied classes, we will have to remove all of them and do refactoring of imports.